src/scaffoldmaker/meshtypes/meshtype_3d_stomach1.py

```diff
@@ -0,0 +1,2473 @@
+                'Vessel wall thickness': 5.0,
+                'Vessel angle 1 degrees': 0.0,
+                'Vessel angle 1 spread degrees': 0.0,
+                'Vessel angle 2 degrees': 0.0,
+                'Use linear through vessel wall': True,
+                'Use cross derivatives': False,
+                'Refine': False,
+                'Refine number of elements around': 4,
+                'Refine number of elements along': 4,
+                'Refine number of elements through wall': 1
+            },
+        }),
+        'Rat 1': ScaffoldPackage(MeshType_3d_ostium1, {
+            'scaffoldSettings': {
+                'Number of vessels': 1,
+                'Number of elements across common': 2,
+                'Number of elements around ostium': 8,
+                'Number of elements along': 2,
+                'Number of elements through wall': 1,  # not implemented for > 1
+                'Unit scale': 1.0,
+                'Outlet': False,
+                'Ostium diameter': 4.0,
+                'Ostium length': 3.5,
+                'Ostium wall thickness': 0.5,
+                'Ostium inter-vessel distance': 0.0,
+                'Ostium inter-vessel height': 0.0,
+                'Use linear through ostium wall': True,
+                'Vessel end length factor': 1.0,
+                'Vessel inner diameter': 1.25,
+                'Vessel wall thickness': 0.5,
+                'Vessel angle 1 degrees': 0.0,
+                'Vessel angle 1 spread degrees': 0.0,
+                'Vessel angle 2 degrees': 0.0,
+                'Use linear through vessel wall': True,
+                'Use cross derivatives': False,
+                'Refine': False,
+                'Refine number of elements around': 4,
+                'Refine number of elements along': 4,
+                'Refine number of elements through wall': 1
+            },
+        }),
+    }
+
+    @staticmethod
+    def getName():
+        return '3D Stomach 1'
+
+    @staticmethod
+    def getParameterSetNames():
+        return [
+            'Default',
+            'Human 1',
+            'Rat 1']
+
+    @classmethod
+    def getDefaultOptions(cls, parameterSetName='Default'):
+        if 'Rat 1' in parameterSetName:
+            centralPathOption = cls.centralPathDefaultScaffoldPackages['Rat 1']
+            ostiumOption = cls.ostiumDefaultScaffoldPackages['Rat 1']       
+        else:
+            centralPathOption = cls.centralPathDefaultScaffoldPackages['Human 1']
+            ostiumOption = cls.ostiumDefaultScaffoldPackages['Human 1']
+
+        options = {
+            'Central path': copy.deepcopy(centralPathOption),
+            'Number of elements around esophagus': 8,
+            'Number of elements around duodenum': 12,
+            'Number of elements between annulus and duodenum': 6,
+            'Number of elements through wall': 1,
+            'Number of radial elements in annulus': 1, 
```

Comment:
When set to 2 this does some funny stuff on the lesser curvature for the rat ... can you look into. May be an issue in annulus mesh which can be fixed later. May want to add an issue on github for it?
Should this be ``Number of elements across cardia``? User doesn't know which annulus you're referring to.